packages/dai/contracts/networks.js

```diff
@@ -138,15 +140,8 @@ export default [
   { name: 'mainnet', networkId: 1, contracts: contractInfo('mainnet') },
   { name: 'morden', networkId: 2 },
   { name: 'ropsten', networkId: 3 },
-  {
-    name: 'rinkeby',
-    networkId: 4
-  },
-  {
-    name: 'goerli',
-    networkId: 5
-  },
-
+  { name: 'rinkeby', networkId: 4 },
+  { name: 'goerli', networkId: 5 },
```

Comment:
had to look this up for myself, I didn't realize some testnets were just incrementing the network Id